src/my_project/tagging/tagging_controller.py

```
@@ -76,123 +76,203 @@ def cover_getter(self):
+
+            total = 0.2 * hash_score + 0.8 * hist_score
+            results.append((index, hash_score, hist_score, total))
+            logger.info(
+                f"idx={index} hash={hash_score:.3f} hist={hist_score:.3f} total={total:.3f}"
+            )
+
+        sorted_scores = sorted(results, key=lambda x: x[3], reverse=True)
+        if len(sorted_scores) == 1:
+            if sorted_scores[0][3] > 0.75:
+                return [sorted_scores[0][0]]
+            else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)